test/functional/apps/dashboard/saved_search_embeddable.ts

```diff
@@ -48,7 +48,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       const marks = $('mark')
         .toArray()
         .map((mark) => $(mark).text());
-      expect(marks.length).to.above(10);
+      expect(marks.length).to.above(0);
```

Comment:
Just out of curiosity, what causes the difference here? Has the highlighting functionality changed?

Reply:
Well, the Document explorer uses virtualization to display just the docs that are visible to the user, not rendering others. So when using document explorer, the expected number is 6. I didn't change it to this number, because I implemented it in a way that it also would work if we decide to step back (because this feature already has a history, of putting it off again).
The test works now for classic and document explorer, and I think not the number of found highlighted terms is relevant, but that there are highlighted terms rendered. Which proves: highlighting works. And if we ever decide to migrate `classic table` -> `document explorer` -> 🍪   ... it should also work